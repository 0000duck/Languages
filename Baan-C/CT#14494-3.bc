choice.test.123:
before.choice:
    funct1(10)
on.choice:
    funct1(10)

before.program:
    set.synchronized.dialog("fxinh0123s000")

functions:
function func1(long var1)
{
    | Do Something 1.
}
