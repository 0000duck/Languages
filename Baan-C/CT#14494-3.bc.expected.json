{
    "root":"CT#14494-3.bc",
    "branches":[
        {
            "name":"choice.test.123"
            "leaves":[
                "before.choice",
                "on.choice"
            ]
        },
        {
            "name":"functions"
            "leaves":[
                "func1"
            ]
        }
    ]
)